### PR TITLE
update techempower port

### PR DESCRIPTION
**Pull request by kusumachalasani**
Signed-off-by: kusumachalasani <kchalasa@redhat.com>

### CODE REVIEW

File: techempower/manifests/quarkus-resteasy-hibernate.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ spec:
   type: NodePort
   ports:
     - port: 8080
-      nodePort: 32334
+      nodePort: 32434
```
Comment (dinogun) [inline techempower/manifests/quarkus-resteasy-hibernate.yaml:53]: Why does this need to be a fixed port number? We can get the port number as we do in the demo [script](https://github.com/kruize/autotune-demo/blob/main/minikube_demo_setup.sh#L260).

Comment (kusumachalasani) [reply techempower/manifests/quarkus-resteasy-hibernate.yaml]: Removed the fixed number. openshift runs doesn't require port. 
Will update the required port changes in minikube PR #43.

File: techempower/manifests/quarkus-resteasy-hibernate.yaml [hunk 2]
```diff
@@ -50,7 +50,6 @@ spec:
   type: NodePort
   ports:
     - port: 8080
-      nodePort: 32334
```
Comment (dinogun) [inline techempower/manifests/quarkus-resteasy-hibernate.yaml:53]: Can we remove this from the petclinic yaml as well

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**